code/game/objects/items/wall_mounted.dm

```diff
@@ -64,6 +64,10 @@
 	// For camera-building borgs
 	var/turf/T = get_step(get_turf(user), user.dir)
 	if(iswallturf(T))
+		if((locate(/obj/machinery/power/apc) in get_turf(user)) && (istype(src, /obj/item/wallframe/apc)))
+			var/obj/machinery/power/apc/mounted_apc = locate(/obj/machinery/power/apc) in get_turf(user)
+			mounted_apc.attackby(src, user)
+			return TOOL_ACT_TOOLTYPE_SUCCESS
```

Comment:
Would this be better to be on ``/obj/item/wallframe/apc/screwdriver_act`` instead?

Reply:
unsure

Reply:
I moved it to apc frame and had it override the wallframe screwdriver proc, tested it and everything works, let me know if im missing anything else.